google/cloud/spanner/CMakeLists.txt

```diff
@@ -78,8 +79,11 @@ function (spanner_client_define_tests)
     # the GTest::gmock target, and the target names are also weird.
     find_package(GTest CONFIG REQUIRED)
 
-    set(spanner_client_unit_tests sql_statement_test.cc spanner_version_test.cc
-                                  value_test.cc)
+    set(spanner_client_unit_tests
+        row_test.cc
+        sql_statement_test.cc
```

Comment:
nit: I would like to keep these alphabetical, swap `spanner_version_test.cc` and `sql_statement_test.cc`

Reply:
Done.